src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -229,7 +229,11 @@ async function minikubeStatus(context: Context): Promise<MinikubeInfo> {
     const result = await context.shell.exec(`"${context.binPath}" status`);
 
     if (result && result.stderr.length === 0) {
-        const hostStatus = result.stdout.split('\n')[0].split(': ')[1].toLowerCase();
+        const hostItem = result.stdout.split('\n').find((status) => status.includes('host'));
+        if (!hostItem) {
+            throw new Error(`Failed to get host status: Unable to run Minikube`);
```

Comment:
I don't think we can put this message into the status function - it is used in places other than the start function.  Also not sure about throwing rather than returning it through the `MinikubeInfo.message` property.